modules/influxdb/influxdb_test.go

```diff
@@ -48,6 +50,29 @@ func TestV2Container(t *testing.T) {
 	}
 }
 
+func TestWithWaitStrategy(t *testing.T) {
+	ctx := context.Background()
+	influxDbContainer, err := influxdb.Run(ctx,
+		"influxdb:2.7.5-alpine",
+		influxdb.WithDatabase("foo"),
+		influxdb.WithUsername("root"),
+		influxdb.WithPassword("password"),
+		testcontainers.WithWaitStrategy(wait.ForHTTP("/health").
+			WithStatusCodeMatcher(func(status int) bool {
+				return status == http.StatusOK
+			})),
+	)
+	testcontainers.CleanupContainer(t, influxDbContainer)
+	require.NoError(t, err)
+
+	state, err := influxDbContainer.State(ctx)
+	require.NoError(t, err)
+
+	if !state.Running {
+		t.Fatal("InfluxDB container is not running")
+	}
```

Comment:
suggestion: we should simplify this with a require
```suggestion
	require.True(t, state.Running)
```